builtins/target-avx-utils.ll

```diff
@@ -294,25 +294,25 @@ define_avgs()
 define(`convert_scale_to_const', `
 
 
- switch i32 %$10, label %default_$1 [ i32 1, label %on_one_$1
+ switch i32 %argn(`10',$@), label %default_$1 [ i32 1, label %on_one_$1
                                       i32 2, label %on_two_$1
```

Comment:
Here and in other instances below - need to change alignment of line following the changed line.

Reply:
Fixed